README.md

```diff
@@ -335,11 +335,11 @@ To release a new version, run either of these commands:
     rake bump:minor
     rake bump:major
 
-This will: update `lib/asana/version.rb`, commit and tag the commit. Then you
+This will: update `lib/asana/version.rb` and `VERSION`, commit and tag the commit. Then you
```

Comment:
Super minor detail, what do you think about making it `/lib/asana/version.rb` and `/VERSION` (with or without a preceding `.`) so that it's clearer that they're in the root directory (especially for `VERSION`)?